core/src/main/java/com/linecorp/armeria/common/util/CompositeException.java

```diff
@@ -155,12 +171,19 @@ public synchronized Throwable getCause() { // NOPMD
                             aggregateMessage.append(separator);
                         }
 
-                        for (int i = 0; i < depth + 2; i++) {
-                            aggregateMessage.append("  ");
-                        }
                         final StackTraceElement[] st = inner.getStackTrace();
                         if (st.length > 0) {
-                            aggregateMessage.append("at ").append(st[0]).append(separator);
+                            final boolean isEnableVerboseExceptionFlag =
+                                    verboseExceptionFlag.isSampled(inner.getClass());
+                            final int loopCount = isEnableVerboseExceptionFlag ?
+                                                  st.length : Math.min(
+                                                          disabledVerboseExceptionLoopCount, st.length);
```

Comment:
Would it be better to give more meaningful names to the variables?
- `isEnableVerboseExceptionFlag`: Grammatically incorrect.
- `loopCount`: Looks general and does not tell the current context
- `disabledVerboseExceptionLoopCount`: Difficult to understand the context
```suggestion
                            final int maxStackTraceSize = isVerboseException ?
                                                  st.length : Math.min(
                                                          DEFAULT_MAX_NUM_STACK_TRACES, st.length);
```

Reply:
thanks for review :) i will fix it